sdcm/utils/decorators.py

```diff
@@ -237,6 +238,8 @@ def wrapped(*args, **kwargs):  # noqa: PLR0914
                 workload = 'write'
             elif 'mixed' in test_name:
                 workload = 'mixed'
+            elif '_throughput' in test_name:
```

Comment:
your'e already overriding `workload_name` in params, so it could be handled by the next `elif` clause. Why it's needed?